protocol/operation.rei

```diff
@@ -2,14 +2,6 @@ open Helpers;
 module Main_chain: {
   [@deriving (ord, yojson)]
   type t =
```

Comment:
`type t` probably needs a better name. Or `Main_chain` needs a doc string.

Reply:
closing as this is not related to this PR